apps/ui/src/pages/dashboard.tsx

```diff
@@ -53,6 +59,16 @@ const DashboardPage = () => {
 		pageSize: 100,
 	});
 
+	/**
+	 * Will auto redirect DAC Members and Institutional Reps to
+	 * their correct routes on component mount.
+	 */
+	useEffect(() => {
+		if (userInfo.isLoggedIn && userInfo.role !== 'APPLICANT') {
+			navigate('/login/redirect', { replace: true });
+		}
+	}, [navigate, userInfo.isLoggedIn, userInfo.role]);
```

Comment:
Shouldn't this be handled in a router file? this kind of logic hidden in the page itself may be hard to trackdown in the future.